tests/test_integration.py

```diff
@@ -0,0 +1,54 @@
+from dotenv import load_dotenv, find_dotenv
+import requests
+import os
+import pytest
+from unittest.mock import patch, Mock
+
+import app.app as app
+from tests.mock_response import MockResponse
+
+
+@pytest.fixture
+def client():
+    # Use our test integration config instead of the 'real' version.
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+
+    # Create the new app.
+    test_app = app.create_app()
+
+    # Use the app to create a test_client that can be used in our tests.
+    with test_app.test_client() as client:
+        yield client
+
+@pytest.fixture
+def mock_response(monkeypatch):
+    monkeypatch.setattr(requests, "get", mock_get)
+
+def mock_get(url, *args, **kwargs):
+    board_id = os.environ.get('BOARD_ID')
+
+    if url == f'https://api.trello.com/1/boards/{board_id}/lists':
+        samples_trello_lists_response = [
+            {'id': '5efc77d5ce13a25c8b6bf9c1', 'name': 'To Do', 'closed': False, 'pos': 65535, 'softLimit': None, 'idBoard': '5efc773fd08d053db4ef3c18', 'subscribed': False},
```

Comment:
Really minor, but it feels like we might as well have the 'idBoard' field match our test board id here